### PR TITLE
Update `GraphClientFactory` using in README.md

### PR DESCRIPTION
**Pull request by gerundt**
I tried to get the `GraphClientFactory` examples to run but seems outdated.

I worked with the updated examples.

### CODE REVIEW

**Review by andrueastman — APPROVED** [review]
Thanks for the contribution here @gerundt!

@Ndiritu for final final review/merge.

File: README.md [hunk 1]
```diff
@@ -71,11 +70,11 @@ class UsageExample
 
         $config = [
             'headers' => [
-                'Authorization' => 'Bearer '.$accessToken
+                'Authorization' => $accessToken
```
Comment (Ndiritu) [inline README.md:73]:
```suggestion
                'Authorization' => 'Bearer '.$accessToken
```
Would the request be successful without the bearer prefix?

**Review by Ndiritu — COMMENTED** [review]
Thanks for the changes @gerundt!

File: README.md (hunk 1 above)
Comment (gerundt) [reply README.md:73]: At least I couldn't get it to work with the keyword. 🤷‍♂️

I read the tip to leave out the `Bearer` somewhere at stackoverflow.com. But I can't find the source at the moment. 🙈

It worked for me after removing the keyword!

Comment (Ndiritu) [reply README.md:73]: Interesting. I'm able to get it working with the "prefix" and without. I'll resolve this for now.

**Review by Ndiritu — APPROVED** [review]

**Pull request merged**